README.md

```diff
@@ -10,4 +10,34 @@ This script is intended to be used for specific needs:
 
 Although EdgeRouter X has [build-in Dynamic DNS client](https://help.ui.com/hc/en-us/articles/204952234-EdgeRouter-Built-in-Dynamic-DNS), it does not work in an environment where its interface does not have global IPv4 address.
 
+## Setup
+Fill out in the header of the script
+
+- auth_email: The email used to login 'https://dash.cloudflare.com
+- auth_key: Top right corner, "My profile" > "Global API Key"
+- zone_identifier: Can be found in the "Overview" tab of your domain on the right side on the lower half of the page
+- record_name: Domain on which you want to update the A record
+
+Copy the script with (win)scp to /config/scripts of your router.
+
+Make it executable: `sudo chmod 0755 /config/scripts/cloudflare-ddns.sh`
+
+Schedule task:
+```
+configure
+set system task-scheduler task cloudflare-ddns executable path /config/scripts/cloudflare-ddns.sh
+set system task-scheduler task cloudflare-ddns interval 1m
```

Comment:
I feel that 1 min interval is too much. When considering burden on external APIs, I think that the default value should be longer (i.e. 15min)

Reply:
Would it help to poll DNS except the API (e.g. dig +short @8.8.8.8), 15 minutes to detect a change seems a bit long to me.